### ISSUE TITLE
Maybe it would be better to switch source of Wine Wayland driver directly to its developer's Gitlab repo for having always automatically updated patches?

### ISSUE THREAD

**Issue by Kerrung** [Frogging-Family/wine-tkg-git#425]
It's here: https://gitlab.collabora.com/alf/wine/-/commits/wayland

[Wine Wayland driver patch in your repo](https://github.com/Frogging-Family/community-patches/blob/master/wine-tkg-git/wine_wayland_driver.mypatch) has an older version than this repository.

That is, I propose to make it so that if wine_wayland_driver.mypatch is added to the _community_patches = "" list in .cfg, the source code of this patch is not downloaded from https://github.com/Frogging-Family/community-patches/blob/master /wine-tkg-git/wine_wayland_driver.mypatch, and instead downloaded directly from https://gitlab.collabora.com/alf/wine/-/commits/wayland. Is it possible to do this? Thank you.

**Comment by Sukid** [comment]
That would be the bees knees. Although I've been trying to make a more updated patch myself and the patch application keeps failing, even if I use plain wine stable. Then again I'm new to this and it might take more than a git format-patch origin --stdout > wayland.patch to make it usable. Tho I also tried using a local copy of the repo (ensuring checked out the Wayland branch) under __localcopy in wine-tkg.cfg and that doesn't seem to compile right either. Always stops with a `../wine-alf/dlls/jscript/parser.y: warning: 1 shift/reduce conflict`

**Comment by hurricane-dorian** [comment]
Not really, the repo has code commits not patches. The patch must  be crated from the commits.
Anyways I've updated the wayland patch and I aim to update it every 2 weeks going forward.
